README.md

```diff
@@ -40,20 +40,20 @@ Please specify a bucket.
 Please specify a region.
 > us-east-1
 Please specify an org.
-> Jemurai
+> tempuslabs
 Please specify a working directory.
 > ~/Desktop/s3s2/
 Please specify a file prefix (nothing sensitive).
-> jemurai_
+> tempuslabs_
 Please specify a public key to use (file path or url).
-> https://s3s2.jemurai.com/.well_known/s3s2-pub.asc
+> https://s3s2.tempuslabs.com/.well_known/s3s2-pub.asc
```

Comment:
This will change once TechOps is done with creating the key pair and host the public key via an url